src/main/java/org/fundacion/jala/converter/core/facade/ChecksumFacade.java

```diff
@@ -39,8 +39,8 @@ private ChecksumFacade() {
      * @return object of ParameterOutputChecksum.
      * @throws IOException is a exception when invalid file's path.
      */
-    public static ParameterOutputChecksum getChecksum(final String checksum,
-                                                      final MultipartFile file) throws IOException {
+    public static ParameterOutputChecksum getChecksum(final String checksum, final MultipartFile file)
+            throws  IOException {
```

Comment:
I think there is an extra space, maybe you could delete it.

Reply:
Solved.